btc-lsp/test/ServerSpec.hs

```diff
@@ -73,7 +73,7 @@ spec = forM_ [Compressed, Uncompressed] $ \compressMode -> do
                               )
                          & Proto.port
                            .~ ( defMessage
-                                  & Proto.val .~ 9735
+                                  & Proto.val .~ 9736
```

Comment:
Internally all lnd p2p ports work on 9735, 9736 and 9737 are only used if access to them is needed by apps outside of the cluster